### DISCUSSION TITLE
setup.bin (created by MeshCentral) doesnt appear to "activate" the network layer

### DISCUSSION THREAD

**Discussion by GRIFFCOMMca** [Ylianst/MeshCentral#6598] [General]
Hi, created a setup.bin file from MeshCentral using ACM and to deactive CCM and CIRA to connect to server.  The setup.bin works when installed in to the client PC, tested with AMT v9 (asked if to configure with USB via BIOS message) and AMT v15 (didnt ask, just took the file and did it).  However it didnt seem to then "activate" the network layer, it did configure it, shouldnt it then be telling AMT to activate it so its working and able to be used by MeshCentral?

**Comment by si458** [comment]
have u made sure the AMT network is set as DHCP, same with the OS?
i discovered a BUG with AMT itself recently where if you set AMT to STATIC, it doesnt activate properly using CIRA

one other thing to test/check is can u login using the meshcommander to the AMT and see settings?
you can use the meshcommander.msi or i recently discovered meshcmd includes it too!
`meshcmd.exe meshcommander` then access using localhost:3000 in your web browser

**Reply by GRIFFCOMMca** [reply]
AMT was DHCP, the whole configuration worked, it just did not activate the network layer, ergo never came online until i manually did that with CTRL+P then manually activated it, seems odd the setup.bin didnt instruct it to do this, but did configure it (as it had the correct new password).

I have another PC with VPro thats not configured, will see if i can test with that (meshcommander) OR use another i can reset as a test unit.  Is meshcommander a seperate install or is it installed with the mesh agent?

**Reply by si458** [reply]
i have no clue how the setup.bin works, ive tried looking for docs/info about it but no look

meshcommander is a seperate application BUT its also included in the meshcmd.exe which i only just discovered!
so download the meshcmd.exe from the web ui, then run `meshcmd.exe meshcommander` and then visit `http://localhost:3000`
you can then add the device as localhost with its admin and amt password and see if you can connect
if not then u might need to active from the mbex as you had to do manually

but im still confused why it didnt activate? maybe an AMT bios bug?

**Reply by GRIFFCOMMca** [reply]
Ylian might be the only person to know this... i too have not be able to find anything on the setup.bin, considering though it did configure ACM, i would assume it would then activate the layer itself, thats the whole point of the setup.bin, a USB key hands of setup.  I'll see if i can do more digging on the MeshCentral and setup bin creation as it might just be a configuration issue.  We had it work the same on AMT v9 and AMT v15, so i dont believe its a bug, more likely something missing in the setup.bin file.

**Reply by Tschak77** [reply]
Hi @GRIFFCOMMca have you found anything. I have same issue with v12 and v19
